magento2-nginx/fpm-7.0/usr/local/share/env/30-framework

```diff
@@ -128,3 +128,7 @@ export MAGENTO_RUN_BUILD
 export MAGENTO_ADMIN_USERNAME=${MAGENTO_ADMIN_USERNAME:-}
 export MAGENTO_ADMIN_PASSWORD=${MAGENTO_ADMIN_PASSWORD:-}
 export MAGENTO_ADMIN_EMAIL=${MAGENTO_ADMIN_EMAIL:-admin@example.com}
+
+MAGENTO_CACHE_STATIC_ASSETS=${MAGENTO_CACHE_STATIC_ASSETS:-false}
+MAGENTO_CACHE_STATIC_ASSETS="$(convert_to_boolean_string "$MAGENTO_CACHE_STATIC_ASSETS")"
```

Comment:
btw, though I'm not going to hold up this PR about this, but I've been converting these convert_to_boolean_string defaults to be less verbose e.g.:
```
MAGENTO_CACHE_STATIC_ASSETS="$(convert_to_boolean_string "${MAGENTO_CACHE_STATIC_ASSETS:-false")"
```